v2.0/ping/README.md

```diff
@@ -0,0 +1,122 @@
+# Ping
+
+This is the simpest example for Singularity Compose to generate two containers
+with an alpine base, and then shell into one to ping the other.
+You'd first want to build the containers:
+
+```bash
+$ singularity-compose build
+```
+
+Note that since this example uses fakeroot for networking, you likely would need
+to update your configuration for your user as follows:
+
+```bash
+$ sudo singularity config fakeroot --add $USER
+```
+
+The above command would add _your_ username. You'll also need to update the fakeroot
+network configuration at `/usr/local/etc/singularity/network/40_fakeroot.conflist`
+(or replace with the prefix where you installed Singularity):
+
+```json
+{
+    "cniVersion": "0.4.0",
+    "name": "fakeroot",
+    "plugins": [
+        {
+            "type": "bridge",
+            "bridge": "sbr0",
+            "isGateway": true,
+            "ipMasq": true,
+            "ipam": {
+                "type": "host-local",
+                "subnet": "10.22.0.0/16",
+                "routes": [
+                    { "dst": "0.0.0.0/0" }
+                ]
+            }
+        },
+        {
+            "type": "firewall"
+        },
+        {
+            "type": "portmap",
+            "capabilities": {"portMappings": true},
+            "snat": true
+        }
+    ]
+}
+``` 
+
+See [here](https://sylabs.io/guides/3.7/user-guide/cli/singularity_config_fakeroot.html) for
+more details. Otherwise, you can remove the "start" sections and the client
+will ask you for sudo. Once this is done, bring them up (debug is used to show
+commands):
+
+```bash
+$ singularity-compose --debug up -d
+Creating alp1
+DEBUG singularity instance start --bind /home/vanessa/Desktop/Code/singularity-compose-examples/v2.0/ping/resolv.conf:/etc/resolv.conf --bind /home/vanessa/Desktop/Code/singularity-compose-examples/v2.0/ping/etc.hosts:/etc/hosts --fakeroot --net --network-args "portmap=1025:1025/tcp" --network-args "IP=10.22.0.2" --hostname alp1 --writable-tmpfs /home/vanessa/Desktop/Code/singularity-compose-examples/v2.0/ping/alp1/alp1.sif alp1 
+WARNING: Disabling --writable-tmpfs as it can't be used in conjunction with underlay
+
+DEBUG singularity exec --env-file=myvars.env instance://alp1 printenv SUPERHERO
+PANCAKEMAN
+
+Creating alp2
+DEBUG singularity instance start --bind /home/vanessa/Desktop/Code/singularity-compose-examples/v2.0/ping/resolv.conf:/etc/resolv.conf --bind /home/vanessa/Desktop/Code/singularity-compose-examples/v2.0/ping/etc.hosts:/etc/hosts --fakeroot --net --network-args "portmap=1026:1026/tcp" --network-args "IP=10.22.0.3" --hostname alp2 --writable-tmpfs /home/vanessa/Desktop/Code/singularity-compose-examples/v2.0/ping/alp2/alp2.sif alp2 
+WARNING: Disabling --writable-tmpfs as it can't be used in conjunction with underlay
+
+DEBUG singularity run  instance://alp2 
```

Comment:
double space here:
`run  instance`

Reply:
This I'll need to fix in the other pr!

Reply:
Commented on the other PR - this is because the run args are empty, and since there isn't an issue with the actual command, I don't see it's super important to remove.